src/inference/src/compilation_context.cpp

```diff
@@ -46,10 +46,12 @@ static int32_t as_int32_t(T v) {
 std::string NetworkCompilationContext::calculateFileInfo(const std::string& filePath) {
     uint64_t seed = 0;
     auto absPath = filePath;
-    try {
-        absPath = FileUtils::absoluteFilePath(filePath);
-    } catch (...) {
-        // can't get absolute path, will use filePath for hash
+    if (filePath.size() > 0) {
+        try {
+            absPath = FileUtils::absoluteFilePath(filePath);
+        } catch (...) {
```

Comment:
should we also catch only specific exception? It throws `std::runtime_error`

Reply:
The purpose of this code seems to be to proceed even though it doesn't get the abs path properly. If so, wouldn't it be okay not to specify a specific exception type?

Reply:
I suppose we need to catch only expected exception, and here we expect only std::runtime_error. If we ignore all other exception, we can miss actual errors in logic and code will work, but not as we expected.

Reply:
I updated to catch only std::runtime_error as you reviewed. Thank you!